### PR TITLE
i18n(pt-BR): Big review and `Aside` component

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description
Updates to almost all PT-BR pages using the most recent PT-BR Translation Guide changes. This PR also adds the new Aside component to the pages.

### CODE REVIEW

File: src/pages/pt-BR/guides/integrations-guide.md [hunk 1]
```diff
@@ -120,7 +121,7 @@ Há três formas comuns de importar uma integração em seu projeto Astro:
 // astro.config.mjs
 import {defineConfig} from 'astro/config';
 import integracaoInstalada from '@astrojs/vue';
-import integracaoLocal from './minha-integracao.js';
+import integracaoLocal from './minha-integracao';
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/integrations-guide.md:124]:
```suggestion
import integracaoLocal from './minha-integracao.js';
```

File: src/pages/pt-BR/guides/markdown-content.md [hunk 2]
```diff
@@ -185,7 +191,7 @@ Você pode importar componentes no seu arquivo Markdown com `setup` e usá-los j
 layout: ../layouts/LayoutBase.astro
 setup: |
   import Autor from '../../components/Autor.astro'
-  import Biografia from '../components/Biografia.jsx'
+  import Biografia from '../components/Biografia'
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/markdown-content.md:194]:
```suggestion
  import Biografia from '../components/Biografia.jsx'
```

File: src/pages/pt-BR/guides/publish-to-npm.md [hunk 3]
```diff
@@ -176,7 +182,7 @@ O **ponto de entrada principal do pacote** é utilizado sempre que seu pacote é
 ```js
 export { default as MeuComponenteAstro } from './MeuComponenteAstro.astro';
 
-export { default as MeuComponenteReact } from './MeuComponenteReact.jsx';
+export { default as MeuComponenteReact } from './MeuComponenteReact';
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/publish-to-npm.md:185]:
```suggestion
export { default as MeuComponenteReact } from './MeuComponenteReact.jsx';
```

File: src/pages/pt-BR/core-concepts/astro-components.md [hunk 4]
```diff
@@ -55,51 +55,53 @@ Você pode utilizar o script do componente para escrever qualquer código JavaSc
...
 Se você escrever puro HTML aqui, o seu componente irá renderizar esse HTML em qualquer página Astro em que é importado e utilizado.
 
-Porém, a sintaxe do template do script do Astro também suporta **expressões do JavaScript**, **componentes importados** e [**diretivas especiais do Astro**](/pt-BR/reference/directives-reference/). Dados e valores definidos (em tempo de construção da página) no script de componente podem ser utilizados no template do script para produzir HTML criado dinamicamente.
+Porém, a sintaxe do template do script do Astro também suporta **expressões JavaScript**, **componentes importados** e [**diretivas especiais do Astro**](/pt-BR/reference/directives-reference/). Dados e valores definidos (em tempo de build da página) no script de componente podem ser utilizados no template do script para produzir HTML criado dinamicamente.
 
 ```astro
 ---
 // O script do seu componente está aqui!
-import ComponentePokemonReact from '../components/ComponentePokemonReact.jsx';
+import ComponentePokemonReact from '../components/ComponentePokemonReact';
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/astro-components.md:86]:
```suggestion
import ComponentePokemonReact from '../components/ComponentePokemonReact.jsx';
```

**Review by victor0x16 — CHANGES_REQUESTED** [review]
My 2 cents !

File: src/pages/pt-BR/core-concepts/framework-components.md [hunk 5]
```diff
@@ -111,34 +115,44 @@ import MeuComponenteVue from '../components/MeuComponenteVue.vue';
...
+
 ## Aninhando Componentes de Frameworks
 
 Dentro de um componente Astro, você também pode aninhar componentes de múltiplos frameworks.
 
 ```astro
 ---
 // src/pages/MinhaPaginaAstro.astro
-import MinhaBarraLateralReact from '../components/MinhaBarraLateralReact.jsx';
+import MinhaBarraLateralReact from '../components/MinhaBarraLateralReact';
```
[the remark this answers is not in the document]
Comment (Yan-Thomas) [reply src/pages/pt-BR/core-concepts/framework-components.md:129]: It's recommended to leave the jsx/tsx file extension from the import now, also this is reflecting the English docs

**Comment by Yan-Thomas** [comment]
Thanks for the review @victor0x16, since the remaining suggestions is to revert something from the English docs, I'm assuming it is okay for me to merge it now.

File: src/pages/pt-BR/core-concepts/framework-components.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (victor0x16) [reply src/pages/pt-BR/core-concepts/framework-components.md:129]: @Yan-Thomas , they are still using `.jsx` extensions. I looked at it just now.

**Comment by delucis** [comment]
Woo Yan merging things! 🥳 

One note re those `.jsx` extensions: we actually haven‘t removed them in the English docs. I opened a PR doing that but we then realised that soon _including_ them will be recommended so we’re sticking with imports with extensions in the docs (although both currently work).

**Pull request merged**